sdk/servicebus/service-bus/src/queueClient.ts

```diff
@@ -195,7 +210,11 @@ export class QueueClient implements Client {
    * It cannot be `Completed/Abandoned/Deferred/Deadlettered`. The lock on it cannot be renewed.
    *
    * @param [maxMessageCount] The maximum number of messages to peek. Default value `1`.
-   * @returns Promise<ReceivedSBMessage[]>
+   * @returns Promise<ReceivedMessageInfo[]>
+   * @throws {Error} Thrown if the underlying connection or client is already closed.
+   * @throws {MessagingError} Thrown if the underlying operation results in an error
+   * from the service. This contains the error code indicating more details on reason
```

Comment:
`MessagingError` doesnt have a `code` property on it. If you want to call attention to any of the properties it should be `name` and `retryable`. Or skip doing so altogether.

![image](https://user-images.githubusercontent.com/16890566/68446387-62575e00-0191-11ea-8dd6-43021b466686.png)

Reply:
Updated

Reply:
Looks like we have decided to remove calling attention to any of the properties of the `MessagingError` 

Logged #6137 for a follow up for improving how we communicate the usage of the `MessagingError` class